components/brave_wallet/browser/zcash/zcash_orchard_storage.cc

```diff
@@ -120,9 +162,40 @@ bool ZCashOrchardStorage::CreateSchema() {
                            " ("
                            "account_id TEXT NOT NULL PRIMARY KEY,"
                            "account_birthday INTEGER NOT NULL,"
-                           "latest_scanned_block INTEGER NOT NULL,"
-                           "latest_scanned_block_hash TEXT NOT NULL);") &&
+                           "latest_scanned_block INTEGER,"
+                           "latest_scanned_block_hash TEXT);") &&
+         database_.Execute(
+             "CREATE TABLE " kShardTree
+             " ("
+             "account_id TEXT NOT NULL,"
+             "shard_index INTEGER NOT NULL,"
+             "subtree_end_height INTEGER,"
+             "root_hash BLOB,"
+             "shard_data BLOB,"
+             "CONSTRAINT shard_index_unique UNIQUE (shard_index, account_id),"
+             "CONSTRAINT root_unique UNIQUE (root_hash, account_id));") &&
+         database_.Execute("CREATE TABLE " kShardTreeCheckpoints
+                           " ("
+                           "account_id TEXT NOT NULL,"
+                           "checkpoint_id INTEGER PRIMARY KEY,"
+                           "position INTEGER)") &&
+         database_.Execute("CREATE TABLE " kCheckpointsMarksRemoved
+                           " ("
+                           "account_id TEXT NOT NULL,"
+                           "checkpoint_id INTEGER NOT NULL,"
+                           "mark_removed_position INTEGER NOT NULL,"
+                           "FOREIGN KEY (checkpoint_id) REFERENCES "
+                           "orchard_tree_checkpoints(checkpoint_id)"
+                           "ON DELETE CASCADE,"
+                           "CONSTRAINT spend_position_unique UNIQUE "
+                           "(checkpoint_id, mark_removed_position, account_id)"
+                           ")") &&
+         database_.Execute("CREATE TABLE " kShardTreeCap
+                           " ("
+                           "account_id TEXT NOT NULL,"
+                           "cap_data BLOB NOT NULL)") &&
          transaction.Commit();
+  // TODO(cypt4): Add indexes
```

Comment:
Is there an issue for this TODO?

Reply:
https://github.com/brave/brave-browser/issues/43529